test_scripts/Resumption/InteriorVehicleData/002_InteriorVD_resumption_one_moduleType_with_moduleId_unexpected_disconnect.lua

```diff
@@ -0,0 +1,51 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description: Successful resuming of interior vehicle data after transport disconnect
+--  in case GetInteriorVehicleData was requested with single module and with moduleId
```

Comment:
@GetmanetsIrina I would propose next rephrasing of the test script to avoid confuse
```suggestion
--  in case app was subscribed on one module with moduleId
```